### PR TITLE
Cleanup/round5

### PR DESCRIPTION
**Pull request by tammalee**
- fixed contact us button
- fixed pagination alignment on mobile
- added scroll bar to table overflow
- added return to top styling on articles
- added more padding to hero image on mobile
- Readme.md updated

### CODE REVIEW

File: styles/_article.scss [hunk 1]
```diff
@@ -205,14 +205,9 @@
 );
 
   a {
-    display: block;
-    padding: 20px 0;
+    display: inline-block;
+    margin: 20px 0;
```
Comment (eddysims) [inline styles/_article.scss]: Can we make this rem?

File: styles/_hero.scss [hunk 2]
```diff
@@ -1,26 +1,18 @@
 /***** Hero component *****/
 .hero {
   width: 100%;
+  padding: 30px 15px;
```
Comment (eddysims) [inline styles/_hero.scss]: rem?

File: styles/_hero.scss [hunk 3]
```diff
@@ -1,26 +1,18 @@
...
-  $max-width,
-  $spacing,
-  $spacing--desktop
-);
+  @media screen and (min-width: 376px) {
+    @include fluid-value( height,
+    $min-width,
+    $max-width,
+    400px,
+    550px
```
Comment (eddysims) [inline styles/_hero.scss]: rem?

File: styles/_hero.scss [hunk 4]
```diff
@@ -29,9 +21,11 @@
 }
 
 .hero-inner {
-  @include vertical-align;
   max-width: 750px;
   margin: 0 auto;
+  @media screen and (min-width: 375px) {
```
Comment (eddysims) [inline styles/_hero.scss]: rem

File: styles/_mid-page-cta.scss [hunk 5]
```diff
@@ -51,6 +51,9 @@
       padding-left: $spacing--desktop;
       text-align: right;
     }
+    @media screen and (min-width: 768px) and (max-width: 787px) {
```
Comment (eddysims) [inline styles/_mid-page-cta.scss]: this shouldn't be a thing. You should be able to do one or the other.

File: styles/_article.scss (hunk 1 above)
Comment (tammalee) [reply styles/_article.scss]: No REM in this one, only EMs. I can look at doing that?

**Pull request merged**